by-language/python-sqlalchemy/async_streaming.py

```diff
@@ -0,0 +1,174 @@
+"""
+About
+=====
+
+Example program to demonstrate how to connect to CrateDB using its SQLAlchemy
+dialect, and exercise a few basic examples using the low-level table API, this
```

Comment:
You can remove `this time`, it's better text to be independent of the read flow, so just `in asynchronous mode` I think it's enough.

Reply:
Thanks. I've addressed all items like you suggested, modulo the `trace()` topic.